devutils/git-hooks/pre-commit

```diff
@@ -15,6 +15,7 @@ source "$(dirname "${BASH_SOURCE[0]}")/../lib/bash/prompt.bash"
 PreCommit::GoChecks () {
     [ -z "$GOPATH" ] && Log::Failure "please install golang on workstation (\$GOPATH missing)"
 
+  exit 0
```

Comment:
Please fix the indentation. Can you also move the next comment onto the line (with two space separating the 0 and the leading `#`)?